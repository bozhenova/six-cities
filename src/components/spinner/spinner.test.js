import React from 'react';
import renderer from 'react-test-renderer';
import Spinner from './spinner';

describe('Spinner component renders correctly', () => {
  it('should render Spinner', () => {
    const tree = renderer.create(<Spinner />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
