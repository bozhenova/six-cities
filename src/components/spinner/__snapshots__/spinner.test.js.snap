// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Spinner component renders correctly should render Spinner 1`] = `
<div
  className="center"
>
  <div
    className="loadingio-spinner-double-ring-06ffj4bbadfx"
  >
    <div
      className="ldio-bq9h1bvhxtj"
    >
      <div />
      <div />
      <div>
        <div />
      </div>
      <div>
        <div />
      </div>
    </div>
  </div>
</div>
`;
